### PR TITLE
build: pin serde and rmp-serde

### PR DESCRIPTION
**Pull request by jost-s**
After the conductor API [breakage due to an interplay between `rmp-serde` and a patch update in `serde`](https://github.com/serde-rs/serde/issues/2662), these two packages are pinned in the Holochain repo. Lair is using a different version of `rmp-serde`, which is why I'd like to match the one in Holochain to prevent serialization problems between Lair and Holochain.

### CODE REVIEW

**Review by neonphog — CHANGES_REQUESTED** [review]
@jost-s - As a library, I'd really prefer if these weren't pinned. If everybody followed semver properly, it "should" be possible to set rmp-serde to `0.15` and leave serde itself at `1`, then let holochain do the actual pinning.

Would you mind testing if that works before we merge this? If it doesn't it could be a problem with the lair binary, in which case we could try doing the pinning only in the `lair_keystore` crate, and leave the deps more flexible in the `lair_keystore_api` crate.

**Comment by jost-s** [comment]
Not at all, great suggestion! Works as it should, ready for review again.

**Review by neonphog — APPROVED** [review]
woot

**Pull request merged**
